README.md

```diff
@@ -0,0 +1,476 @@
+  * [Table of Contents](#table-of-contents)
+  * [Changelog](#changelog)
+  * [Contributors](#contributors)
+  * [To-Do](#to-do)
+  * [Standard](#standard)
+    + [R000 - Recommended Software](#r000---recommended-software)
+    + [R001 - Universite de Sherbrooke Computer Science Department Programming Standard](#r001---universite-de-sherbrooke-computer-science-department-programming-standard)
+    + [R002 - Project Repository Structure](#r002---project-repository-structure)
+    + [R003 - Pythonic - The Zen of Python - PEP 20](#r003---pythonic---the-zen-of-python---pep-20)
+    + [R004 - Main](#r004---main)
+    + [R005 - Paths - Working Directory](#r005---paths---working-directory)
+    + [R006 - Style - PEP 8](#r006---style---pep-8)
+    + [R007 - Naming Variables - Functions - Classes](#r007---naming-variables---functions---classes)
+    + [R008 - Type Hinting - PEP 484](#r008---type-hinting---pep-484)
+    + [R009 - Docstring - PEP 257 - Google Style](#r009---docstring---pep-257---google-style)
+    + [R010 - Comments](#r010---comments)
+    + [R011 - String Quotes](#r011---string-quotes)
+    + [R012 - f-strings](#r012---f-strings)
+    + [R013 - Multiprocessing and Compilation](#r013---multiprocessing-and-compilation)
+    + [R014 - Enumerations - PEP 435](#r014---enumerations---pep-435)
+    + [R015 - Abstract Classes - PEP 3119](#r015---abstract-classes---pep-3119)
+    + [R016 - Encapsulation - Private vs Protected vs Public](#r016---encapsulation---private-vs-protected-vs-public)
+    + [R017 - Decorators](#r017---decorators)
+
+NOTES: 
+
+- To update the Table of Contents, use: https://ecotrust-canada.github.io/markdown-toc/
+- Section headers cannot contain special characters other than -, otherwise the TOC hyperlinks will not work
+
+## Changelog
+
+Revision | Date       | Description |
+---------| -----------| ----------- |
+C        | 2021-10-04 | Added new items in to-do list    |
+B        | 2021-09-13 | Updated based on team's comments    |
+A        | 2021-08-08 | Creation    |
+
+## Contributors
+
+- [Simon Giard-Leroux](https://github.com/sgiardl)
+
+## To-Do
+
+- [x] Table of contents
+- [x] Paths
+- [x] More details on variable names (ie. equation variable), max chars of variable name
+- [x] Enums
+- [x] ABC
+- [x] What is pythonic? List comprehension, etc.
+- [x] Constants (all caps) vs Environment Variables
+- [x] Multiprocessing
+- [x] Check existing udes code standard & integrate contents
+- [x] if __name__ == '__main__'
+- [x] Encapsulation : protected vs private
+- [x] Reorder rules in a logic way
+- [ ] Class inheritance vs aggregation
+- [ ] decorators : staticmethod classmethod abstractmethod
+- [ ] Function/method calls : specify 1 or multiple arguments per line?
+- [ ] Use 'pipreqs' package to generate project 'requirements.txt' file
+- [ ] Start file and folder names in repository with lower case letters
+- [ ] Bare * in arguments list to force use of keyword arguments and prevent positional arguments
+
```

Comment:
j'ajouterais peut-être une section sur les 'is'  (identity), '==' (equality), les 'Falsy/Truthy' vs 'True/False' et autres affaires puisqu'il a souvent confusion dans ces concepts.

Reply:
https://www.freecodecamp.org/news/truthy-and-falsy-values-in-python/